message-queue-lib/src/cmr/message_queue/queue/memory_queue.clj

```diff
@@ -56,33 +57,38 @@
       (when-not (= :done (first (a/alts!! (vec channels) :default :done)))
         (recur)))))
 
-(defrecord MemoryQueueBroker
-  [
-   ;; A list of queue names
-   queues
+(defrecord
+ MemoryQueueBroker
+ [;; A list of queue names
+  queues
 
    ;; A map of exchange names to sets of queue names to which the exchange will broadcast.
-   exchanges-to-queue-sets
+  exchanges-to-queue-sets
 
    ;; A map of queue names to core async channels containing messages to deliver
-   queues-to-channels
+  queues-to-channels
 
    ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
    ;; Running State
 
-   ;; An atom containing a sequence of channels returned by the go block processors for each handler.
-   handler-channels-atom]
+   ;; An atom containing a sequence of channels returned by the go block processors
+   ;; for each handler.
+  handler-channels-atom]
 
 
   ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
   lifecycle/Lifecycle
 
+  ;; param 1 is 'this'
+  ;; param 2 is system
   (start
-    [this system]
+    [this _]
```

Comment:
you could also use _system and then get rid of the ;; param 1 is this ...

Reply:
that will totally work! I did not know that this worked the way you showed